css.css

```diff
@@ -0,0 +1,486 @@
+/*
+colors: 
+Gunmetal = 022F40
+Pacific Blue = 38AECC
+Blue NCS = 0090C1
+Gunmetal = 183446
+Blue Sapphire = 046e8f
```

Comment:
please sort selectors by comments
for example:
`/*________menu_______  */
under this comment add tags only for **menu**

`/*________footer_______  */

under this comment add tags only for **footer**

etc....